File: result/translate/transcoder/java2python/python/FIND_EXPRESSION_DUPLICATE_PARENTHESIS_NOT.py
def f_gold ( s ) :
    Stack = Stack ( )
    str = s.split ( )
    for ch in str :
        if ch == ')' :
            top = Stack.pop ( )
            Stack.pop ( )
            elementsInside = 0
            while top != '(' :
                elementsInside += 1
                top = Stack.pop ( )
                Stack.pop ( )
            if elementsInside < 1 :
                return True
        else :
            Stack.push ( ch )
    return False

File: result/translate/starchat/python2java/java/FIND_EXPRESSION_DUPLICATE_PARENTHESIS_NOT.java
public static boolean f_gold ( String string ) {
        Stack < Character > Stack = new Stack < > ( );
        for ( int i = 0 ; i < string. length ( ) ; i ++ ) {
            char ch = string. charAt ( i );
            if ( ch == ')' ) {
                char top = Stack. pop ( );
                int elementsInside = 0;
                while ( top!= '(' ) {
                    elementsInside ++;
                    top = Stack. pop ( );
                }
                if ( elementsInside < 1 ) return true;
            } else Stack. push ( ch );
        }
        return false;
    }